### ISSUE TITLE
Disabled Ipv6 (in kernel) leads to err: open /proc/sys/net/ipv6/conf/all/forwarding: no such file or directory

### ISSUE THREAD

**Issue by KarstenB** [kubevirt/kubevirt#10857]
**What happened**:
I upgraded Kubevirt to latest version and now VMs no longer start on my machine. They get stuck in starting state with:
`failed to configure vmi network: setup failed, err: open /proc/sys/net/ipv6/conf/all/forwarding: no such file or directory`

**What you expected to happen**:
They work just as before

**How to reproduce it (as minimally and precisely as possible)**:
Add ipv6.disable=1 to your kernel bootcmdline

**Additional context**:
No.

**Environment**:
- KubeVirt version (use `virtctl version`): 1.1.0
- Kubernetes version (use `kubectl version`): k3s v1.27.5+k3s1
- OS (e.g. from /etc/os-release): Debian GNU/Linux 12 (bookworm
- Kernel (e.g. `uname -a`): 6.1.0-13-amd64

**Comment by matthewei** [comment]
have you set the `sysctl -w net.ipv6.conf.all.forwarding=1`

**Comment by EdDev** [comment]
@KarstenB , please provide the vm/vmi manifest and the relevant logs.

I think we are covering only IPv6 single stack in tests.

**Comment by matthewei** [comment]
yes, could you share your vm's yaml? I would like to verify it in order to make sure that your env is healty.

**Comment by KarstenB** [comment]
Just for clarification, when I remove the kernel argument all VMs work fine. 
```
# Create namespace
apiVersion: v1
kind: Namespace
metadata:
  name: virtualmachines
---
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata:
  name: "qemu"
  namespace: "virtualmachines"
  labels:
    kubevirt.io/os: debian-12
spec:
  running: true
  template:
    metadata:
      labels:
        kubevirt.io/size: small
        kubevirt.io/domain: "qemu"
    spec:
      domain:
        devices:
          blockMultiQueue: true
          disks:
            - disk:
                bus: virtio
              name: containervolume
            - disk:
                bus: virtio
              name: cloudinitvolume
          interfaces:
          - name: default
            bridge: {}
        cpu:
            cores: 4
            sockets: 1
            threads: 8
        resources:
          requests:
            memory: 16Gi
            cpu: 4
          limits:
            memory: 16Gi
            cpu: 8
      networks:
      - name: default
        pod: {}
      terminationGracePeriodSeconds: 0
      volumes:
        - name: containervolume
          dataVolume:
            name: qemu-dv
        - name: cloudinitvolume
          cloudInitNoCloud:
            userData: |-
              #cloud-config
              # ... our init data
  dataVolumeTemplates:
  - metadata:
      name: qemu-dv
    spec:
      pvc:
        accessModes:
        - ReadWriteOnce
        resources:
          requests:
            storage: 20Gi
      source:
        http:
          url: https://cdimage.debian.org/cdimage/cloud/bookworm/20231013-1532/debian-12-generic-amd64-20231013-1532.qcow2
```

**Comment by matthewei** [comment]
Actually, the vm's yaml works well in my kubevirt.

**Comment by EdDev** [comment]
> @KarstenB , please provide the vm/vmi manifest and the relevant logs.

Please also share the full logs of the relevant virt-launcher and/or virt-handle pods where you saw the error message.

**Comment by hanqingwu** [comment]
@KarstenB , i have trace the debug, and i found root cause in here. 
![image](https://github.com/kubevirt/kubevirt/assets/7521141/99c7a17e-8068-4d5a-b123-4258da822112)
it read "/proc/sys/net/ipv6/conf/all/forwarding"  , but your env disable ipv6 . so  got  a  error  msg.
maybe i can submit a PR  to fix it .

**Comment by EdDev** [comment]
Fixed on v1.1 & v1.2.

/close
